src/main/java/frc/robot/vision/AprilTagCamera.java

```diff
@@ -130,7 +130,7 @@ public AprilTagCamera(
 
     poseEstimator =
         new PhotonPoseEstimator(
-            Vision.fieldLayout, PoseStrategy.MULTI_TAG_PNP_ON_COPROCESSOR, robotToCamTransform);
+            Vision.fieldLayout, PoseStrategy.LOWEST_AMBIGUITY, robotToCamTransform);

```

Comment:
We switch to multitag before worlds (maybe states), and I was disappointed in it.  It really gets messed up when the field drifts over the events and the gaps between tags really changes.  This makes HUGE deltas in position, and I think it was why our autos at worlds sucked.